package.json

```diff
@@ -47,37 +49,57 @@
     "underscore": "1.4.4 - 1.8.3"
   },
   "devDependencies": {
-    "chai": "1.9.1",
+    "babel-core": "^5.2.17",
+    "babel-eslint": "^4.0.5",
+    "babelify": "^6.0.0",
+    "browserify": "^11.0.1",
+    "chai": "^3.2.0",
     "chai-jq": "0.0.7",
     "chai-jquery": "1.2.1",
+    "del": "^1.1.1",
     "dox": "git://github.com/jasonLaster/dox.git#marked",
+    "esperanto": "^0.7.4",
     "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-clean": "0.5.0",
-    "grunt-contrib-concat": "0.1.2",
-    "grunt-contrib-connect": "0.1.2",
-    "grunt-contrib-jshint": "0.10.0",
-    "grunt-contrib-uglify": "0.2.0",
-    "grunt-contrib-watch": "0.6.1",
-    "grunt-coveralls": "0.3.0",
-    "grunt-env": "0.4.1",
-    "grunt-istanbul": "0.3.0",
-    "grunt-jscs": "^1.5.0",
-    "grunt-lintspaces": "0.7.0",
-    "grunt-mocha-test": "0.10.2",
-    "grunt-plato": "0.1.4",
-    "grunt-preprocess": "4.0.0",
-    "grunt-template": "0.2.3",
+    "grunt-babel": "^5.0.1",
```

Comment:
Darn. Thought I got 'em all. Had to put some of the old deps back cause boilerplate overwrote them. Will change.